### ISSUE TITLE
[Feature request] Resolve %name% when executing commands

### ISSUE THREAD

**Issue by jortvanleenen** [Ajneb97/Codex#4]
Hi,

Could you add %name% resolving to command execution, just like you are currently doing for %player%? E.g. I have a discovery with attribute: 'name: "&a&ltest"'. I hope that %name% would then result in the unformatted text 'test'. Thanks in advance!

**Comment by Ajneb97** [comment]
What do you want to do exactly, can you give me an example?

**Comment by jortvanleenen** [comment]
An example would be 'warp %name% %player%', or '%player% add permission <some-permissions>.%test%' (pseudo example)

**Comment by jortvanleenen** [comment]
Of course, it is very well possible to hardcode the correct value for the command, but I basically need to add the same commands to every discovery and inventory item and would like it to dynamically update based on the name, so it would make the process less cumbersome and prone to errors :)
